cmd/utils.go

```diff
@@ -2049,8 +2049,7 @@ func downloadFileToDisk(log *LoggingConfig, url string, destFile string, dryrun
 }
 
 // tar and zip a directory into .tar.gz
-func Targz(log *LoggingConfig, source, target string) error {
-	filename := filepath.Base(source)
+func Targz(log *LoggingConfig, source, target, filename string) error {
 	log.Info.log("source is: ", source)
 	log.Info.log("filename is: ", filename)
 	log.Info.log("target is: ", target)
```

Comment:
Do we need the above lines to be logged to the console as part of the command. Shouldn't this be debug info?

Reply:
Yes, as long as we leave the printed path of where it has put the archived files, that should be fine, I agree with all the lines It can look very cluttered.